virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -0,0 +1,120 @@
++
+You create an SSH key pair, add the public key to a VM, and connect to the VM by running the `virtctl ssh` command with the private key.
+
+* xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#virt-using-openssh-and-virtctl-port-forward_virt-accessing-vm-ssh[`virtctl port-forward` command]
++
+You add the `virtctl port-foward` command to your `.ssh/config` file and connect to the VM by using OpenSSH.
+
+* xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#using-services-ssh_virt-accessing-vm-ssh[Service]
++
+You create a service, associate the service with the VM, and connect to the IP address and port exposed by the service.
+
+* xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#using-secondary-networks-ssh_virt-accessing-vm-ssh[Secondary network]
++
+You configure a secondary network, attach a virtual machine (VM) to the secondary network interface, and connect to the DHCP-allocated IP address.
+
+include::modules/virt-access-configuration-considerations.adoc[leveloffset=+1]
+
+[id="using-virtctl-ssh_virt-accessing-vm-ssh"]
+== Using virtctl ssh
+
+You can add a public SSH key to a virtual machine (VM) and connect to the VM by running the `virtctl ssh` command.
+
+This method is simple to configure. However, it is not recommended for high traffic loads because it places a burden on the API server.
+
+[id="adding-public-keys-vm_virt-accessing-vm-ssh"]
+=== Adding an SSH key to a virtual machine
+
+You can add a _static_ or _dynamic_ public SSH key to a virtual machine (VM) by using the {product-title} web console or the command line.
+
+Static key::
+* Added to the VM at startup.
+* Uses the cloud-init configuration drive and does not affect cloud-init user data.
+* Can be added when you create a VM by using the xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#adding-static-public-key-vm-web_virt-accessing-vm-ssh[web console] or the xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#adding-static-public-key-project-web_virt-accessing-vm-ssh[command line].
+* Can be added to a xref:../../virt/virtual_machines/virt-accessing-vm-ssh.adoc#adding-static-public-key-project-web_virt-accessing-vm-ssh[project] by using the web console. Afterwards, the key is automatically added to the VMs that you create in this project.
+
+Dynamic key::
+* Can be added or revoked while the VM is running.
+* Can be disabled for security reasons. If the setting is disabled, the VM inherits the key setting of the image from which it was created.
+* Uses the QEMU guest agent.
+* Only supported by {op-system-base-full} 9.
+
+include::modules/virt-adding-static-public-key-vm-web.adoc[leveloffset=+3]
+
+include::modules/virt-adding-static-public-key-cli.adoc[leveloffset=+3]
+
+include::modules/virt-adding-static-public-key-project-web.adoc[leveloffset=+3]
+
+// dynamic ssh key configuration will go here
+
+include::modules/virt-running-virtctl-ssh-command.adoc[leveloffset=+2]
+
+.Copying the `virtctl ssh` command
```

Comment:
Nit: I haven't seen this type of trailing include.  A `[TIP]` admonition with a trailing `.Additional resources` section might be a bit more standard.